### DISCUSSION TITLE
weeklies/5

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#143] [Comments]
https://hutusi.com/articles/weekly-5 

 本文为周刊第五期，但距离前一期已经过去半年。叫周刊似乎不合适，但又想到更好的名字，暂且用这个名字，只不过会不定期发布。前四期: 周刊开刊, 周刊第二期, 周刊第三期, 周刊第四期。

**Comment by robertu7** [comment]
"Why Privacy Is the Most Important Concept of Our Time" 的鏈接指錯了 XD

**Comment by hutusi** [comment]
@robertu7 谢谢指出^_^。已 fix~~
